dev/map.html

```diff
@@ -9,7 +9,6 @@
 </head>
 <body>
 <script type="module">
-  import "@vaadin/map/enable";
```

Comment:
Let's exclude this from this PR and submit it separately so that we can cherry-pick to `23.2` branch.

Reply:
Done